automation/vars/uni01alpha.yaml

```diff
@@ -0,0 +1,34 @@
+---
+vas:
```

Comment:
Typo? Should be vars?

Reply:
VA's... There is a patch in flight and waiting for it to be merged before I start unit testing.